src/store/metadata/stub/content/datasets.json

```diff
@@ -267,8 +267,53 @@
               "modified": "2022-08-18T10:51:00Z"
             }
           ]
+      },
+      {
+        "@id": "https://staging.idpd.uk/datasets/DLUHC",
+        "@type": "dcat:DatasetSeries",
+        "identifier": "existing_dwellings_data",
```

Comment:
Change identifier to match the slug of the `@id`, so `epcs-for-existing-domestic-properties`

Reply:
changed identifier as `epcs-for-existing-domestic-propertie`